src/components/organisms/RichMenu/RichMenuListContent/RichMenuListContent.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from "@components/atoms";
+import { RichMenuList } from "@components/molecules/RichMenulist";
+import { ChevronRightIcon } from "@heroicons/react/24/outline";
+import { RichMenuListContentProps } from "./type";
+
+export const RichMenuListContent = ({ listGroup }: RichMenuListContentProps) => {
+  return (
+    <div className="my-4 flex max-h-[400px] w-fit flex-col gap-y-2 border-l border-gray-extraLight px-4">
+      <div className="flex flex-1 gap-x-4 overflow-hidden rounded bg-gray-extraLight p-4">
+        {listGroup.map((list, index, passedArray) => (
+          <>
+            <RichMenuList
+              {...list}
+              key={list.title}
+            />
+            {index < passedArray.length - 1 && <div className="h-full w-px bg-gray-light"></div>}
+          </>
```

Comment:
The `key` should always be on the outer element. In this case it would be the Fragment. And the key should always be unique, will that be the case with `list.title`?
```suggestion
        {listGroup.map((list, index, passedArray) => (
          <React.Fragment key={list.title}>
            <RichMenuList
              {...list}
            />
            {index < passedArray.length - 1 && <div className="h-full w-px bg-gray-light"></div>}
          </React.Fragment>
```